mysql-test/suite/rpl/include/rpl_heartbeat_zero_timestamp.inc

```diff
@@ -0,0 +1,97 @@
+################################################################################
+# Testing the second behind master is correct with timestamp in HB event
+################################################################################
+
+source include/not_valgrind.inc;
+source include/not_parallel.inc;
+
+# Start a new master-slave
+--source include/have_binlog_format_row.inc
+--source include/master-slave.inc
+--source include/rpl_set_gtid_mode.inc
+--source include/have_debug.inc
```

Comment:
Move to the top (please also merge the not_foo block with this one while at that)